specification/paths/Returns-v1-reasons.json

```
@@ -20,6 +20,15 @@
     "parameters": [
       {
         "$ref": "#/components/parameters/query-filter-tags"
+      },
+      {
+        "in": "header",
+        "name": "Accept-Language",
+        "schema": {
+          "type": "string"
+        },
+        "description": "Description of your optional header",
+        "required": false
```

```suggestion
        "description": "Determines the language in which the ``description`` is returned,"
```
and maybe something about "ISO 3166-1 alpha-2 compliant language tag" like you mentioned below

---

done